core/src/main/scala/com/salesforce/op/dsl/RichTextFeature.scala

```diff
@@ -235,6 +235,8 @@ trait RichTextFeature {
       hashSpaceStrategy: HashSpaceStrategy = TransmogrifierDefaults.HashSpaceStrategy,
       defaultLanguage: Language = TextTokenizer.DefaultLanguage,
       hashAlgorithm: HashAlgorithm = TransmogrifierDefaults.HashAlgorithm,
+      detectSensitive: Boolean = false,
```

Comment:
may make sense to have an enum of treatment modes instead of adding many booleans. 
`SensitvieFeatureMode = { DETECT_ONLY, DETECT_AND_REMOVE}`